docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -115,9 +115,9 @@ Preparing For a Release
       # The following command means that you will not be asked whether
       # you want to continue connecting
       ssh-keyscan github.com >> ~/.ssh/known_hosts
-      git clone git@github.com:ClusterHQ/flocker.git "flocker-${VERSION}"
-      cd flocker-${VERSION}
-      mkvirtualenv flocker-release-${VERSION}
+      git clone git@github.com:ClusterHQ/flocker.git
+      cd flocker
+      mkvirtualenv flocker-release
       pip install --editable .[release]
       admin/create-release-branch --flocker-version="${VERSION}"
```

Comment:
:question: While doing the 1.0.0+doc1 release I wondered if would ever matter that you're install release tools from the master branch and expecting them to work on the release branch...

So this is the master version of `create-release-branch`...not much we can do about that, but....